### ISSUE TITLE
Streaming responses for Mistral models are getting truncated

### ISSUE THREAD

**Issue by ihmaws** [langchain-ai/langchain-aws#215]
**Description**
When using any of the Mistral models in streaming mode, the last token will get truncated if a stop sequence is added.

**Reproducible Sample**

```python
from langchain_aws import ChatBedrock
from langchain_core.messages import HumanMessage, SystemMessage

messages = [
    SystemMessage(
        content="Respond in no more than 5-8 words."
    ),
    HumanMessage(
        content="Hello, how are you?"
    )
]

llm = ChatBedrock(
    model_id="mistral.mistral-small-2402-v1:0",
    streaming=True,
    model_kwargs={
        "stop": ["test"] #needed to trigger the bug; this causes Bedrock service to add last token alongside the stop_reason
    },
)

#print streamed chunks
for chunk in llm.stream(input=messages):
    print(chunk.content, end="", flush=True)
```
  
**Example Truncated Response (Input = _"Hello, how are you?"_)**
_I'm an AI, I don't have feelin_

**Comment by ihmaws** [comment]
Thanks @3coins!

I have a backport to v0.1.18 ready. I can create a PR once a branch to base off is ready: https://github.com/langchain-ai/langchain-aws/compare/v0.1.18...ihmaws:langchain-aws:dev/ihm/fix-mistral-streaming-v0.1.18?expand=1

**Comment by ihmaws** [comment]
Backport into v0.1 here: https://github.com/langchain-ai/langchain-aws/pull/222
